app/src/main/java/com/ucmobiledevelopment/freeways/dto/Incident.kt

```diff
@@ -12,5 +12,5 @@ data class Incident(
     var longitude: String = "",
     var way1: String = "",
     var way2: String = "",
-    var vehiclesInvolved: Int = 0
+    var vehiclesInvolved: Int = 0,
```

Comment:
Trailing commas, while not required, are an easy way to reduce future technical debt: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas